keel-core/src/main/kotlin/com/netflix/spinnaker/keel/actuation/EnvironmentConstraintRunner.kt

```diff
@@ -77,37 +77,46 @@ class EnvironmentConstraintRunner(
     envContext: EnvironmentContext,
     pendingVersionsToCheck: MutableSet<String>
   ) {
-    var version: String? = null
```

Comment:
feels like it wasn't super necessary to change the variable names, only pointing that out because it makes it a bit harder to review the logging changes and confirm that it's logging only.

Reply:
I can't help but make opportunistic improvements when I have a chance... 🤷‍♂️